test/Microsoft.ML.Tests/TrainerEstimators/SdcaTests.cs

```diff
@@ -88,11 +88,66 @@ public void SdcaLogisticRegression()
             Assert.InRange(first.Probability, 0.8, 1);
         }
 
+        [Fact]
+        public void SdcaLogisticRegressionWithWeight()
```

Comment:
>LogisticRegression [](start = 24, length = 18)

This is called `LogisticRegression` but contains `MaximumEntropy` trainers.

Reply:
WIll fix by separating into 2 tests one each for  binary and multiclass 

---
In reply to: [270208235](https://github.com/dotnet/machinelearning/pull/3104#discussion_r270208235) [](ancestors = 270208235)